### PR TITLE
Remove Setup tests and update OSX Peer trust faiures issue

### PR DESCRIPTION
**Pull request by hongdai**
* The set up tests are also fragile because it does not use the same way
we install the certificates on Linux/Osx machines. Fixed them to use a simple scenario to verify the setup.

### CODE REVIEW

**Comment by zhenlan** [comment]
@mconnew, would like to get your opinion of removing the infrastructure test as well.

**Comment by hongdai** [comment]
@dotnet-bot test InnerLoop CentOS7.1 Debug please.

**Comment by hongdai** [comment]
@dotnet-bot test OuterLoop OSX10.12 Release please

**Comment by hongdai** [comment]
@dotnet-bot Test OuterLoop OSX10.12 Release please.

**Comment by hongdai** [comment]
@zhenlan Could you take a look at the updated PR?

File: src/System.Private.ServiceModel/tests/Common/Scenarios/SetupValidationTests.4.1.1.cs [hunk 1]
```diff
@@ -5,64 +5,99 @@
...
-
-        // *** VALIDATE *** \\
-        // Validate rather than allowing an exception to propagate
-        // to be clear the exception was anticipated. 
-        Assert.True(exception == null, exception == null ? String.Empty : exception.ToString());
     }
 
     [WcfFact]
-    [Issue(1913, OS = OSID.AnyUnix)]
     [Issue(1886, OS = OSID.AnyOSX)]
```
Comment (zhenlan) [inline src/System.Private.ServiceModel/tests/Common/Scenarios/SetupValidationTests.4.1.1.cs]: Should this be changed to 1945?

**Comment by zhenlan** [comment]
+@mconnew for code review

File: src/System.Private.ServiceModel/tests/Common/Scenarios/SetupValidationTests.4.1.1.cs (hunk 1 above)
Comment (hongdai) [inline src/System.Private.ServiceModel/tests/Common/Scenarios/SetupValidationTests.4.1.1.cs]: Yes. I'll change it.

**Comment by hongdai** [comment]
@dotnet-bot Test OuterLoop Ubuntu14.04 Please

**Comment by mconnew** [comment]
>         }

Nothing else looks to be using the ValidateCertificate method now. Why not remove that method completely and inline the functionality here?

---
Refers to: src/System.Private.ServiceModel/tests/Common/Scenarios/SetupValidationTests.4.1.1.cs:113 in 6627e0a. [](commit_id = 6627e0a007d07e420626b7f10f5c88bf67f0c1eb, deletion_comment = False)

**Comment by mconnew** [comment]
:shipit:

**Pull request merged**